src/utils/transactionUtils.ts

```diff
@@ -30,6 +31,22 @@ import {
   drop,
 } from "lodash/fp";
 
+const currencySymbols = {
+  USD: "$",
+};
+
+type CurrencySymbolsIndices = keyof typeof currencySymbols;
+
+export const dineroUSD = (amount: number) => dinero({ amount, currency: USD });
+
+export const format = (dineroObject: Dinero<number>, fractionDigits?: number) =>
```

Comment:
@sarahdayan I see that Dinero v2 is in alpha and I'd like to provide some feedback around this DX.

While the flexibility for this `format` method is great, I'm wondering why the need to define such a large callback around a given currency?  This DX feels a bit unintuitive and for a currency's "default", I would expect the library to handle this for me.  Have you've considered including default helpers such as this or simplifying this piece for v2?

Reply:
Hey @kevinold and thanks for the feedback!

You're right, the flexibility comes at the expense of simplicity here. I'm currently rethinking this function for several other reasons, and providing a default return value is part of the thinking process.

The main reason why picking a default is difficult is that Dinero also handles non-decimal currencies. While it might make sense to return "USD 10.50" by default for `dinero({ amount: 1050, currency: USD })`, there's no good option when passing non-decimal objects. More context in [this issue](https://github.com/dinerojs/dinero.js/issues/294).

I've [reworked things](https://github.com/dinerojs/dinero.js/pull/309) and now should be able to expose enough information to `toFormat` to make formatting easier.